spec/support/capybara.rb

```diff
@@ -0,0 +1,47 @@
+require "capybara/rspec"
+
+RSpec.configure do |config|
+  config.use_transactional_fixtures = false
+
+  config.before(:suite) do
+    if config.use_transactional_fixtures?
+      raise(<<~MSG)
+                  Delete line `config.use_transactional_fixtures = true` from rails_helper.rb
+                  (or set it to false) to prevent uncommitted transactions being used in
+                  JavaScript-dependent specs.
+        #{'  '}
+                  During testing, the app-under-test that the browser driver connects to
+                  uses a different database connection to the database connection used by
+                  the spec. The app's database connection would not be able to access
+                  uncommitted transaction data setup over the spec's database connection.
+      MSG
+    end
+
+    DatabaseCleaner.clean_with(:truncation)
```

Comment:
I think we can move these settings to rails_helper because database cleaner is used not only for feature specs. Then we can remove this file